app/views/ht_approval_requests/index.html.erb

```diff
@@ -24,23 +25,23 @@
 
     <% @reqs.each do |req| %>
       <tr <%= raw @added_users.include?(req.userid) || @renewed_users.include?(req.userid) ? 'class="success"' : '' %>>
-        <% if can?(:edit, :ht_approval_requests) %>
-          <td>
-            <% if req.received.present? %>
-                <%= check_box_tag 'ht_users[]', req.userid %>
-              <% end %>
-          </td>
-        <% end %>
         <td>
-          <% # FIXME: first if clause is always true, right? Remove if so. %>
-          <% if HTApprovalRequest.for_approver(req.approver).count.positive? &&
-                can?(:edit, :ht_approval_requests) %>
+          <% if can?(:edit, :ht_approval_requests) %>
             <%= link_to req.approver, edit_ht_approval_request_path(req.approver) %>
           <% else %>
             <%= req.approver %>
           <% end %>
         </td>
-        <td><%= link_to req.userid, ht_user_path(req.userid) %></td>
+        <td>
+          <% if can?(:edit, :ht_approval_requests) %>
```

Comment:
I'd suggest extracting this logic to `HTUserPresenter`

Reply:
Can you explain why the validator required avoiding a standalone checkbox column? I definitely prefer the way it looked before; I don't think this is worth spending a lot of time on, but I at least want to understand what the issue was.

Reply:
The error I get is "form elements must be labeled" (this is using the Firefox dev tools accessibility checker). Using the ht_user e-mail as a label seems better than taking up real estate with a do-nothing label like "Select: " in each row. I can use the e-mail field as a label across table cells, if it's preferable to keep a dedicated "Select" column.

And I agree that the logic belongs in the presenter.

Reply:
I see -- yeah, I agree using the e-mail as a label makes sense; I think it would be preferable to use it as a label across table cells if possible.

Reply:
It does work to go cross-cell for labeling. To my surprise, frankly.